sql: TestLogic: expected 'invalid utf8', but found success

Happened spuriously on my WIP. Unfortunately I'm not able to produce more details, but maybe it can be stressed.

```
--- FAIL: TestLogic (3.99s)
	logic_test.go:1216: testdata/builtin_function:116: expected "invalid utf8", but found success
	logic_test.go:1226: testdata/builtin_function:119: too many errors encountered, skipping the rest of the input
```

--- a/sql/testdata/builtin_function
+++ b/sql/testdata/builtin_function
@@ -112,9 +112,10 @@ SELECT SUBSTR('RoacH', -2, 4)
 ----
 R
 
-# See #6601
+# See #6601 and #8210. We use three arguments to guarantee a very high
+# probability that at least one of the UUIDs is invalid UTF8.
 query error invalid utf8
-select cast(uuid_v4() as string)
+select cast(uuid_v4() as string), cast(uuid_v4() as string), cast(uuid_v4() as string)
 
 query T
 SELECT SUBSTR('12345', 2, 77)

